modules/oe_theme_content_event/src/Plugin/ExtraField/Display/DetailsExtraField.php

```diff
@@ -98,10 +98,12 @@ public function viewElements(ContentEntityInterface $entity) {
         'items' => [
           [
             'icon' => 'file',
+            'size' => 'xs',
```

Comment:
<img width="971" alt="Screenshot 2022-01-07 at 13 14 33" src="https://user-images.githubusercontent.com/1183951/148529140-5da6e589-52b2-41db-8fb5-10838fa8e088.png">

Icons have size `xs` in Full and Teaser views both. Ticket EWPP-1830 asks to change Teaser only.
Looks like icon size should be set in Drupal\oe_theme_content_event\Plugin\ExtraField\Display\TeaserDetailsExtraField and be passed to addRenderableLocation and addRenderableOnlineType as an optional argument.

Reply:
I agree that the ticket says only Teaser but Teaser and Full view modes are using the same pattern for rendering this information, are we sure we want this change only in Teaser?

Reply:
You can pass different size of icons to pattern as an option.
Image from spec: https://webgate.ec.europa.eu/fpfis/wikis/download/attachments/416223976/Event%20CT%20-%[…]esktop.png?version=3&modificationDate=1598608568914&api=v2
Icon as is in full, and small in teaser